inst/catalan.m

```diff
@@ -25,7 +26,7 @@
 %% @example
 %% @group
 %% catalan ()
-%%   @result{} (sym) Catalan
+%%   @result{} (sym) G
```

Comment:
you can workaround this failing use `doctest: +SKIP_UNLESS...`, see `@sym/harmonic` for example